packages/rancher-pushprox/charts/values.yaml

```diff
@@ -16,14 +16,16 @@ global:
   cattle:
     systemDefaultRegistry: ""
 
+namespaceOverride: ""
```

Comment:
Author's comment:
This new variable is added but not used in the monitoring chart in this PR.